README.md

```diff
@@ -121,11 +142,7 @@ CREATE TABLE IF NOT EXISTS :schema.datenintegration
     datum timestamp without time zone NOT NULL,
     amt bigint NOT NULL,
     checksum character varying COLLATE pg_catalog."default",
-    CONSTRAINT datenintegration_pkey PRIMARY KEY (t_id),
-    CONSTRAINT datenintegration_amt_fkey FOREIGN KEY (amt)
```

Comment:
Instead of removing this, I would suggest to add a comment that the constraint "datenintegration_amt_fkey" might raise conflicts if a dataset is replaced by ili2pg and, therefore, a user has to decide if he wants to ommit it.